concepticondata/conceptlists/Luangthongkum-2019-341.tsv

```diff
@@ -195,7 +195,7 @@ Luangthongkum-2019-341-193	193	moon, month	2463	MOON OR MONTH
 Luangthongkum-2019-341-194	194	mother, female (of animals)		
 Luangthongkum-2019-341-195	195	mouse, rat	2139	MUROID (MOUSE OR RAT)
 Luangthongkum-2019-341-196	196	mynah		
-Luangthongkum-2019-341-197	197	nail		
+Luangthongkum-2019-341-197	197	nail	2128	CLAW OR NAIL
```

Comment:
Are we sure there is some CLAW meaning to it? Otherwise, it could be FINGERNAL OR TOENAIL, which is what "nail" is usually mapped to